packages/calcite-components/src/components/stepper/stepper.scss

```diff
@@ -6,27 +8,31 @@
  * @prop --calcite-stepper-action-background-color: defines the background color of an action sub-component inside the component.
  * @prop --calcite-stepper-action-background-color-hover: defines the background color of an action sub-component when hovered or focused.
  * @prop --calcite-stepper-action-background-color-active: defines the background color of an action sub-component when active.
+ * @prop --calcite-stepper-step-bar-fill-color : Specifies fill color for step-bar when `layout="horizontal-single"`.
```

Comment:
Q: Should this tokens be specific ? The reason being that `calcite-stepper-item-border-color` token allows users to specify the border-color at `stepper-item` level.  Adding the above tokens helps users customize them when `layout ="horizontal-single"`. Wonder if its ideal to specify them twice or should this be replaced with `calcite-stepper-item-border-color`.

cc @macandcheese @ashetland @SkyeSeitz

Reply:
One way is to get rid of border-top for stepper-item and let user configure step-bar irrespective of layout.

Reply:
currently it matches, but if the user wants to customize the colors using tokens they need to setup twice once for horizontal & vertical and again for horizontal-single. Both horizontal and vertical layout depend on stepper-item color where as horizontal-single depends on step-bar component to show border like.

Reply:
Sorry for the confusion, i was under the impression that responsive mode is a default setting when the container size decreases but we made a change recently where user needs to specify the layout `horizontal-single` for single stepper view. Make sense to have these separate set of tokens . Ideally one set of tokens would be preferable but the way component is designed can be tricky .

Reply:
I still don’t know if we need two - can we use the same nomenclature for both? The user still places the steps in the DOM in horizontal-single right? I might be missing something too, lol.

Reply:
With the current way the component is design we end up with having one set of tokens to customize border for vertical & horizontal and other set for horizonatal-layout. In horizontal-single layout case the border isn't the border of stepper-item but a functional step-bar which is not attached to any stepper-item.  Using step-bar for all the layout would be ideal but requires a refactor. 


User can customize the border color as following:

``` 
// covers the use-case of layout="horizontal-single"
 calcite-stepper {
      --calcite-stepper-step-bar-fill-color: green;
      --calcite-stepper-step-bar-active-fill-color: #e9f6ff;
      --calcite-stepper-step-bar-complete-fill-color: #280274;
      --calcite-stepper-step-bar-error-fill-color: #fe7a36;
    }
    
    
    
//covers use-case of layout="horizontal" or layout="vertical"

  calcite-stepper-item {
      --calcite-stepper-item-border-color: green;
    }    
    
 calcite-stepper-item[error] {
      --calcite-stepper-item-border-color: #fe7a36;
    }
    
    calcite-stepper-item[complete] {
      --calcite-stepper-item-border-color: #280274;
    }
    
    calcite-stepper-item[selected] {
      --calcite-stepper-item-border-color: #e9f6ff;
    }

Reply:
I think I refactor might be worth it, in the long-term at least. Not sure if this is the time to do that or not. I think this situation may have come about do to limitations of how we build stuff in Figma. That version needs a separate step-bar for `horizontal-single` out of necessity, but I don't see why we'd need to mimic that in the real version.

Reply:
Agree on the refactor, ideally these colors needs to be customized once irrespective of the layout. Configuring the colors twice is redundant and can be very confusing. 

The reason for adding `step-bar` in the real version is to display the borders which are attached to stepper-item. In `horizontal-single` layout mode where only one item is viewed,  borders of each stepper-item is unavailable. To resolve this situation we came up with the step-bar component. In the initial designs we are displaying only one border in single-view related to currently viewing stepper-item. In further discussion, felt it is better to show the step-bar equivalent to no of stepper-items renders so that user has an idea of how many steps are present.  

i think its better to use step-bar for all the layout and drop the border maybe since `horizontal-single` layout cannot rely on border? (considering stepper-item's are not used outside of stepper) . Open to suggestions.

Reply:
How much would a refactor take? This seems like a lot of extra tokens.